src/object/obj_internal.h

```diff
@@ -474,9 +474,10 @@ struct obj_auxi_args {
 					 rebuilding:1,
 					 for_migrate:1;
 	/* request flags. currently only: ORF_RESEND */
-	uint32_t			 flags;
 	uint32_t			 specified_shard;
+	uint16_t			 flags;
```

Comment:
currently we already used more than 16 bits, 
such as -
5572 >       if (args->extra_flags & DIOF_EC_RECOV_FROM_PARITY)                                          
5573 >       >       obj_auxi->flags |= ORF_EC_RECOV_FROM_PARITY; 

ORF_EC_RECOV_FROM_PARITY = (1 << 19),